concrete/src/Page/Collection/Collection.php

```diff
@@ -1113,58 +1041,45 @@ public function addFeature(Feature $fe)
      */
     public function getFeatureAssignments()
     {
-        if (is_object($this->vObj)) {
-            return CollectionVersionFeatureAssignment::getList($this);
-        }
+        $version = $this->getVersionObject();
 
-        return [];
+        return $version === null ? [] : CollectionVersionFeatureAssignment::getList($this);
     }
 
     /**
-     * Update the last edit date/time.
+     * Mark this collection as newly modified.
```

Comment:
What do you mean with 'newly modified'?

Reply:
You know... Spaghetti English... 🤣 
Fixed.